02_transform/extract_methods/contactpoint.py

```diff
@@ -0,0 +1,54 @@
+import json
+
+def entityContactPoint(entity_id, washlist):
+    contactpoints = []
+    contactpoint = {}
+    check_list = contactPointEmail(entity_id)
+    include_name = True
+
+    if check_list in washlist['whiteList']:
+        contactpoint['email'] = contactPointEmail(entity_id)
```

Comment:
bruk check_list i stedet for å hente ut eposten på ny, kan og gi varabelen et bedre navn, som feks email

Reply:
done